### ISSUE TITLE
Lamda dashboard is empty

### ISSUE THREAD

**Issue by pbochynski** [kyma-project/kyma#2541]
**Description**

I created lambda and it works well. I wanted to see some stats for it. I opened grafana dashboard using link provided in lambda list view.

**Expected result**

I wanted to see some data, e.g. number of requests.

**Actual result**

Lambda Dashboard is pretty empty.
<img width="833" alt="screen shot 2019-01-31 at 20 34 42" src="https://user-images.githubusercontent.com/191769/52080522-45ee1b80-2598-11e9-8113-cacca4df8fa3.png">

**Steps to reproduce**

- Install Kyma 0.6.1
- create simple lambda that returns request headers:
```
module.exports = { main: function (event, context) {
  return event.extensions.request.headers
} }
```
- expose lamda with https
- call it several times: 
```
curl https://test.stage.your.kyma.cluster
```
- open dashboard

**Comment by joek** [comment]
Issue is fixed by #2540. Keep the ticket around till test improvements are merged.

**Comment by pbochynski** [comment]
> Issue is fixed by #2540. Keep the ticket around till test improvements are merged.

Thanks for the fix. Until it is merged the workaround would be:
```
kubectl edit servicemonitor istio-mesh -n kyma-system
```
and replace `prometheus: core` with `prometheus: monitoring` in section metadata.labels.
